apps/condo/domains/news/components/TemplatesSelect.tsx

```diff
@@ -34,9 +67,9 @@ export const TemplatesSelect: React.FC<INewsFormProps> = ({ items, onChange, has
     const processedOptions = useMemo(() => {
         if (hasCategories) {
             const result = []
-            for (const category of Object.keys(templatesByCategory)) {
+            for (const category of (Object.keys(templatesByCategory).reverse())) {
```

Comment:
why do you need reverse() here?

Reply:
Now on the site the item 'without template' is displayed last, because now we are adding by using reduce => the default element will be the first, in order for it to work as before, you need to reverse the array